src/tests/mock/http.rs

```diff
@@ -80,6 +82,10 @@ pub enum Expectation {
 pub enum RequestBody {
     /// The received `[RequestBody]` has to match the provided value exactly.
     Exact(serde_json::Value),
+    /// The received `[RequestBody]` has to match the provided value partially
+    /// excluding the specified paths which are represented as dot-separated
+    /// strings.
+    Partial(serde_json::Value, Vec<String>),
```

Comment:
Let's go for simple changes until we actually need something more complex.
```suggestion
    Partial(serde_json::Value, Vec<&'static str>),
```